README.md

```diff
@@ -1,3 +1,135 @@
 # Piko/RT
 
 This is `Piko/RT`, a tiny Linux-like real-time operating system kernel, optimized for ARM Cortex-M series microprocessors.
+
+
+Run STM32-p103 emulator
+-----------------------
+
+You will need to prepare QEMU STM32 emulator from: [QEMU STM32 v0.1.3](https://github.com/beckus/qemu_stm32/releases)
+
+After download, compile the qemu_stm32 by:
+
+```
+$ ./configure --disable-werror --enable-debug \
+        --target-list="arm-softmmu" \
+        --extra-cflags=-DDEBUG_CLKTREE \
+        --extra-cflags=-DDEBUG_STM32_RCC \
+        --extra-cflags=-DDEBUG_STM32_UART \
+        --extra-cflags=-DSTM32_UART_NO_BAUD_DELAY \
+        --extra-cflags=-DSTM32_UART_ENABLE_OVERRUN --python=python2
+$ make -j8
+```
+
+Make sure you have export the `arm-softmmu` to `$PATH`
+
+```
+$ export PATH=$PATH:~/qemu_stm32-stm32_v0.1.3/arm-softmmu
+```
+
+You will need to modify the target from `stm32f429` to
```

Comment:
It is not necessary to modify `Makefile`. Instead, you only have to execute `make TARGET=stm32p103`.